.github/pull_request_template.md

```diff
@@ -17,4 +17,4 @@
 * [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
 * [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
 * [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
-* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.
+* [ ] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".
```

Comment:
```suggestion
* [ ] If this pull request resolves one or more open issues then the commit body (not commit title) includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".
```

Reply:
I worded it as "PR description" rather than "commit body" because I was thinking that we should have the issue number here:
<img width="846" alt="PR text box with issue number" src="https://github.com/TheAlgorithms/Python/assets/52298854/b04f0939-3949-476c-a57b-ac50ddf06054">
because only then does GitHub link the PR itself to the issue. When you put the issue number in the commit title/body but not the PR description, GitHub only links that specific commit [but not the rest of the PR](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue):
> You can also use closing keywords in a commit message. The issue will be closed when you merge the commit into the default branch, but the pull request that contains the commit will not be listed as a linked pull request.

I believe this means that we wouldn't see indications like this:
<img width="326" alt="screenshot 2" src="https://github.com/TheAlgorithms/Python/assets/52298854/deaf6a6a-c567-4718-9a36-191f25e51794">
which would make it harder to track linked issues/PRs.

While it's true that adding the issue number to the commit body means that GitHub can automatically add it to the PR description as well, that doesn't happen if the PR consists of multiple commits.